src/blocks/blocks/stripe-checkout/inspector.js

```diff
@@ -13,14 +13,77 @@ import {
 	SelectControl,
 	Spinner,
 	TextControl,
-	TextareaControl
+	TextareaControl,
+	ExternalLink
 } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
-import { ButtonToggleControl, RichTextEditor } from '../../components/index.js';
-import { useState } from '@wordpress/element';
+import { ButtonToggleControl, Notice as OtterNotice, Notice, RichTextEditor } from '../../components/index.js';
+import { Fragment, useContext, useState } from '@wordpress/element';
+import { applyFilters } from '@wordpress/hooks';
+import { FormContext } from '../form/edit';
+import { setUtm } from '../../helpers/helper-functions';
+
+const ProFeatures = () => {
+	return (
+		<Fragment>
+
+			<PanelBody
+				title={ __( 'Autoresponder (Pro)', 'otter-blocks' ) }
```

Comment:
We can replace the (Pro) part from the title, same for From block. We want people to click here to explore and then find out it's pro.